config/components/local-db/201-sql-deployment.yaml

```diff
@@ -64,6 +64,8 @@ spec:
               - ALL
             add:
               - NET_BIND_SERVICE
+      securityContext:
+        fsGroup: 1001
```

Comment:
I feel this should go in document rather than in default config because this will not work in platform with default restrictions like Openshift.
That can break the tekton operator deployment of results. Correct me if I am wrong @khrm .

Reply:
What default restrictions Openshift have about `securityContext. fsGroup`?

Reply:
I deployed this in Openshift earlier today with default SCC and it didn't bring the pod up. I can check again later.

Reply:
@khrm Does this have any impact on the tekton operator deployment of tekton results in Openshift clusters?